src/server/middleware/security/v2_token.go

```diff
@@ -62,6 +65,11 @@ func (vt *v2Token) Generate(req *http.Request) security.Context {
 		return nil
 	}
 	cl := &v2TokenClaims{}
+	// Add some leeway to claims to account for clock skews on distributed environments
+	// https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4
+	cl.Claims.RegisteredClaims.ExpiresAt = jwt.NewNumericDate(cl.Claims.RegisteredClaims.ExpiresAt.Time.Add(time.Second * 10))
```

Comment:
Hi @an-toine , 
we would like to keep this time frame as same as upstream distribution  which will be 60s.
https://github.com/distribution/distribution/blob/7c354a4b40feeea21d7eeae4de91c8ff7951e672/registry/auth/token/token.go#L25C4-L25C4
Could you help to update this and fix the CI issue ? Appreciate!